src/openms_gui/source/VISUAL/Spectrum3DOpenGLCanvas.cpp

```diff
@@ -180,8 +180,11 @@ namespace OpenMS
 
   void Spectrum3DOpenGLCanvas::initializeGL()
   {
-    makeCurrent();
-    glEnable(GL_DEPTH_TEST);
+    // The following line triggered a bug where the whole screen would turn
+    // black during scrolling (specifically it seems that multiple calls to
+    // this function causes the issue):
+    // glEnable(GL_DEPTH_TEST);
```

Comment:
basically what happened on Linux: when Spectrum3DCanvas.cpp:202 was commented out, then 3D view would work but scrolling was not possible. When not commented out, the scrolling would kindof work but the screen would go black and only if this line `glEnable(GL_DEPTH_TEST);` is commented out then it would all work.